be/src/column/binary_column.h

```diff
@@ -287,11 +287,12 @@ class BinaryColumnBase final : public ColumnFactory<Column, BinaryColumnBase<T>>
         std::stringstream ss;
         ss << "[";
         size_t size = this->size();
-        for (int i = 0; i < size - 1; ++i) {
-            ss << debug_item(i) << ", ";
-        }
-        if (size > 0) {
-            ss << debug_item(size - 1);
+        for (size_t i = 0; i < size; i++) {
```

Comment:
few tips:
- the original code is trying to eliminate branch mis-predication https://en.wikipedia.org/wiki/Predication_(computer_architecture)
- but this is ok if this function not work in the normal query path
- a minor suggestion, you might wanna place condition `i != size-1` before because it will hits the branch predication in most of iteration

Reply:
Original implementation will occur array out of index.

You can assume `size = 0`, `size - 1` will produce a large positive number.